builtin/logical/pki/path_fetch.go

```diff
@@ -203,6 +240,11 @@ func pathFetchListCerts(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "certs/?$",
 
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixPKI,
+			OperationSuffix: "certs",
```

Comment:
Should these have a `list` verb? `pki-list-certs`?

Reply:
The List verb is implied from the List operation below, this is the generated one:


| Path | OperationID |
| ---- | ---------- |
| "/{pki_mount_path}/certs" |        pki-list-certs |